packages/documentation/src/stories/components/card-control/card-control.stories.ts

```diff
@@ -69,7 +80,8 @@ export const Default: Story = {
   render: (args: Args) => {
     const [, updateArgs] = useArgs();
 
-    const icon = html`<span slot="icon">${unsafeHTML(args['slots-icon'])}</span> `;
+    const content = html`<div slot="content">${unsafeHTML(args['slots-content'])}</div>`;
+    const icon = html`<span slot="icon">${unsafeHTML(args['slots-icon'])}</span>`;
```

Comment:
This is right, but since we do not exaclty know what the user is inserting into the slot (which can be done trough the controls table), we are not able to this.